source/minar.cpp

```diff
@@ -251,7 +251,7 @@ int minar::SchedulerData::start(){
 
             if(dispatch_tree.get_num_elements() > 0) {
                 CallbackNode *root = dispatch_tree.get_root();
-                now_plus_tolerance = wrapTime(now + root->tolerance);
+                now_plus_tolerance = wrapTime(now + root->tolerance/2);
```

Comment:
OK, I don't think that playing with the tolerance is a good idea for now. When we fix it, we'll fix it properly. Let's leave the code as it is for now.